### DISCUSSION TITLE
How can I bring back N and L keys to native vim mappings?

### DISCUSSION THREAD

**Discussion by timjackleus** [LazyVim/LazyVim#1578] [Q&A] [answered]
LazyVim is using the keys H and L to cycle between buffers, my questing is how I can remove them.

**Comment by dpetka2001** [comment, marked as answer]
`vim.keymap.del("n", "<S-h>")` and the same for `L` in your `/lua/config/keymaps.lua`

**Reply by timjackleus** [reply]
Perfect, thank you!
